patches/chromium/fix_make_std_hash_value-non-const.patch

```diff
@@ -0,0 +1,21 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Shelley Vohr <shelley.vohr@gmail.com>
+Date: Tue, 20 Aug 2019 09:30:20 -0700
+Subject: fix: make std::hash value-non-const
+
+
+diff --git a/ui/events/blink/prediction/filter_factory.h b/ui/events/blink/prediction/filter_factory.h
```

Comment:
Please add a description for why this change has been made and also link the upstream CL.